docs/WritingTests.asciidoc

```diff
@@ -235,6 +235,11 @@ def run():
     # [...] omitted for brevity
 -------------------------------------------------------------------
 
+Additionally, the `run` method in Python is not able to handle any argument
```

Comment:
According to @perlpunk it is possible to run Perl-defined baseclass-functions on the first argument (`self`), see https://github.com/os-autoinst/openQA/pull/5340/files#r1366928714. So I guess this statement is not true then. The argument is at least handled in the way that it allows calling back Perl-defined functions - and being able to call those functions is likely also generally wanted.

Reply:
Agreed, Will change the wording to reflect better that `self` is the only argument accepted and `run_args` is not supported for Python.